src/merkle/smt/full/mod.rs

```diff
@@ -294,3 +299,66 @@ impl From<&RpoDigest> for LeafIndex<SMT_DEPTH> {
         Word::from(value).into()
     }
 }
+
+// SERIALIZATION
+// ================================================================================================
+
+impl Serializable for Smt {
+    fn write_into<W: ByteWriter>(&self, target: &mut W) {
+        // serialize storage maps
```

Comment:
This comment is outdated, maybe remove it?

Reply:
good catch